Interop/Runtime/src/main/kotlin/kotlinx/cinterop/Utils.kt

```diff
@@ -34,6 +34,8 @@ public object nativeHeap : NativeFreeablePlacement {
     override fun alloc(size: Long, align: Int) = nativeMemUtils.alloc(size, align)
 
     override fun free(mem: NativePtr) = nativeMemUtils.free(mem)
+
+    fun freeAll() = nativeMemUtils.freeAll()
```

Comment:
This undesirably affects cinterop API for Native. And despite its name, the function actually does nothing.
On the other hand, it doesn't seem necessary here, in common part.

Reply:
Ok, extracted to common utils visible for both `JvmNativeMem` and `Context`.